app/src/main/res/layout/viewflipper_setting_ip.xml

```diff
@@ -47,6 +65,17 @@
                 android:scrollHorizontally="true"
                 android:text="http://www.xwiki.org/xwiki"
                 android:textColor="@android:color/black" />
+            <Button
+                android:layout_weight="1"
+                android:layout_width="100dp"
+                android:layout_height="50dp"
+                android:text="Next"
```

Comment:
Do add all the hardcoded values in their respective files. Strings, dimensions, colors etc. 
I know it wasn't managed from starting but adding new code should follow the guidelines.